processing/data_collection/gazette/spiders/pr_maringa.py

```diff
@@ -0,0 +1,48 @@
+from dateparser import parse
+from datetime import date, datetime
+import re
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class PrMaringaSpider(BaseGazetteSpider):
+    TERRITORY_ID = '4115200'
+    name = 'pr_maringa'
+    allowed_domains = ['maringa.pr.gov.br']
+
+    def start_requests(self):
+        """
+        @url http://venus.maringa.pr.gov.br/arquivos/orgao_oficial/paginar_orgao_oficial_ano.php
+        @returns requests 1
+        """
+        todays_date = date.today()
+        current_year = todays_date.year
+        for year in range(current_year, 2006, -1):
+            yield scrapy.FormRequest(
+                'http://venus.maringa.pr.gov.br/arquivos/orgao_oficial/paginar_orgao_oficial_ano.php',
+                headers={
+                    'Content-Type': 'application/x-www-form-urlencoded'
+                },
+                formdata={
+                    'ano': str(year),
+                    'entrar': 'Buscar'
+                },
+                callback=self.parse_year
+            )
+
+    def parse_year(self, response):
+        rows = response.css('table tr')[1:]
+        for row in rows:
+            gazette_id = row.css('td:nth-child(1) a::attr(href)').re_first('.*/[oO]{2}[mM] (.*)')
+            gazette_date = row.css('td:nth-child(2) font > font::text').extract_first()
+            yield Gazette(
+                date=parse(f'{gazette_date}', languages=['pt']).date(),
+                file_urls=[f'http://venus.maringa.pr.gov.br/arquivos/orgao_oficial/arquivos/oom%20{gazette_id}'],
+                is_extra_edition=any(extra_char in gazette_id for extra_char in ['A', 'B', 'C', 'D']),
```

Comment:
Extra edition identification correct based on 2009 gazettes list. The way that it is works, but I do prefer to identify just a letter, not just this 4, just in case of more than 4 extras in a day. What do you think?
Something like: ```is_extra_edition=any(caracter.isalpha() for caracter in gazette_id),```